src/kudu/consensus/raft_consensus.cc

```diff
@@ -1658,12 +1664,14 @@ Status RaftConsensus::UpdateReplica(const ConsensusRequestPB* request,
     // that were actually prepared, and deleting the other ones since we've taken ownership
     // when we first deduped.
     if (iter != messages.end()) {
-      LOG_WITH_PREFIX_UNLOCKED(WARNING) << Substitute(
-          "Could not prepare transaction for op '$0' and following $1 ops. "
-          "Status for this op: $2",
-          (*iter)->get()->id().ShortDebugString(),
-          std::distance(iter, messages.end()) - 1,
-          prepare_status.ToString());
+      if (expected_rotation_delay) {
```

Comment:
Should this be `!expected_rotation_delay`?

Reply:
yep, my bad